roles/tendrl-storage-node/README.md

```diff
@@ -18,6 +18,13 @@ Role Variables
   Specifying this variable is mandatory as there is no default value.
 * Variable `graphite_ip_address` needs to be set to ipv4 adress of graphite
   instance. Specifying this variable is mandatory as there is no default value.
+* When `etcd_authentication` variable is undefined or set to `True` (which is
+  the default value), this role will specify etcd username and password in
+  tendrl config files. When the value is `False`, the authentication will be
+  disabled. For production, keep the authentication always enabled.
```

Comment:
Same as above, already enabled authentication will not be disabled.

Reply:
Ok.

Reply:
Addressed in 39210f7847f53e8eb4e02ba60a581af860fe6e53